cli/cmd/capture/create.go

```diff
@@ -40,12 +40,14 @@ var (
 	hostPath           string
 	includeFilter      string
 	includeMetadata    bool
+	interfaces         string
 	jobNumLimit        int
 	maxSize            int
 	namespace          string
 	namespaceSelectors string
 	nodeNames          string
 	nodeSelectors      string
+	noAllInterfaces    bool
```

Comment:
noAllInterfaces is clunky, use the affirmative and have it be allInterfaces, since that's what the any flag does

and we want the default to be all interfaces via any, only being overridden when the interface flag is set